interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -710,6 +710,40 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  /** Sends an rpc to an unimplemented method on the server. */
+  @Test(timeout = 10000)
+  public void unimplementedMethod() {
+    UnimplementedServiceGrpc.UnimplementedServiceBlockingStub stub =
+        UnimplementedServiceGrpc.newBlockingStub(channel);
+    try {
+      stub.unimplementedCall(Empty.getDefaultInstance());
+      fail();
+    } catch (StatusRuntimeException e) {
+      assertEquals(Status.UNIMPLEMENTED.getCode(), e.getStatus().getCode());
+    }
+  }
+
+  /** Start a fullDuplexCall which the server will not respond, and verify the deadline expires. */
```

Comment:
I don't think the server will not respond, just the client gets deadline exceeded before server responds.

How about just "Start a fullDuplexCall with a short deadline, and verify the stream gets DEADLINE_EXCEEDED".